code/datums/components/itemblock/warmsup.dm

```diff
@@ -0,0 +1,40 @@
+//example component for itemblocks - heats up the user on life tick while blocking
+/datum/component/itemblock/warmsup
```

Comment:
I feel like this is a misnomer. It doesn't just warm you up, it also gives cold protection.